.bazelrc

```diff
@@ -10,9 +10,6 @@ build:asan --linkopt="-fsanitize=address"
 build:asan --linkopt="-fsanitize=leak"
 build:asan --linkopt="-fsanitize-link-c++-runtime"
 
-build:lld --linkopt="-fuse-ld=lld"
```

Comment:
I would keep this if the behavior changes across version. We don't have a line enforcing minimum version in our bazel files.

Reply:
I would regard this Bazel change as a "bugfix". Bazel always tries to find lld for clang, and we need them only because Bazel failed to find ours... Now it is fixed.

Reply:
IMHO it's bad to all-in for the cutting edge feature, even though it's just a bugfix. I tend to keep backward compatibility for a while if it's already there and doesn't really hurt, or not to hard to implement.

Reply:
For example, if you found another bug later on and want to compare this with bazel 4, you'll have to restore this.

Reply:
Right, but those lines are also terrible because it may not work with other llvm versions. `-fuse-ld=lld` alone is not helpful.

Reply:
linker is not as sensitive as compiler. It can often stays for years without breaking compatibility, because ABI rarely changes.